website/cue/reference.cue

```diff
@@ -610,7 +610,7 @@ _values: {
 
 	// `unit` clarifies the value's unit. While this should be included
 	// as the suffix in the name, this helps to explicitly clarify that.
-	unit: #Unit | null
+	unit?: #Unit | null
```

Comment:
Similar to common, we don't always have a unit to provide, such as if the number is dimensionless or it's better to explain the unit in the field description (i.e. explaining the rate of the sample transform).

We also don't want it to always be there simply as null if not present... the field not existing serves the same purpose.